### PR TITLE
add a job_info file to the unit tests

### LINKED ISSUE #139: add job_info to C++ unit tests
We lost this functionality in the port to the C++ build system, so we should add it back

### PR DESCRIPTION
**Pull request by zingale**
This adds a job_info file to all of the unit tests that produce a plotfile.
Closes #139 
Blocked by #380

### CODE REVIEW

**Comment by zingale** [comment]
all tests pass: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2020-08-31-001/index.html

File: unit_test/write_job_info.cpp [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  // plotfile information
+  jobInfoFile << PrettyLine;
+  jobInfoFile << " Plotfile Information\n";
+  jobInfoFile << PrettyLine;
+
+  time_t now = time(0);
+
+  // Convert now to tm struct for local timezone
+  tm* localtm = localtime(&now);
+  jobInfoFile   << "output date / time: " << asctime(localtm);
```
Comment (ChrisDeGrendele) [inline unit_test/write_job_info.cpp:52]: This gives the wrong time for me. `date` in the terminal gives: `Tue 01 Sep 2020 12:17:01 AM EDT`
but in job_info it gives: `output date / time: Tue Sep  1 00:16:27 2020`

Comment (ChrisDeGrendele) [reply unit_test/write_job_info.cpp:52]:
```
time_t now = time(0);                                                                                                                                                                          
char* dt = ctime(&now);                                                                           
jobInfoFile   << "output date / time: " << dt;
```

This works for me and it handles timezones correctly. I checked New York and California. 

The output looks like:
```output date / time: Mon Aug 31 21:41:59 2020 ```

**Comment by ChrisDeGrendele** [comment]
Other than that ^ this looks good.

**Pull request merged**